cpp/cmake_modules/DefineOptions.cmake

```diff
@@ -104,33 +112,36 @@ if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
   define_option(ARROW_USE_PRECOMPILED_HEADERS "Use precompiled headers when compiling"
                 OFF)
 
-  define_option_string(ARROW_SIMD_LEVEL
-                       "Compile-time SIMD optimization level"
-                       "SSE4_2" # default to SSE4.2
-                       "NONE"
-                       "SSE4_2"
-                       "AVX2"
-                       "AVX512")
-
-  define_option_string(ARROW_RUNTIME_SIMD_LEVEL
-                       "Max runtime SIMD optimization level"
-                       "MAX" # default to max supported by compiler
-                       "NONE"
-                       "SSE4_2"
-                       "AVX2"
-                       "AVX512"
-                       "MAX")
+  define_option_string(
```

Comment:
I think we can alter this tabulation behavior using https://cmake-format.readthedocs.io/en/latest/configopts.html#min-prefix-chars

Reply:
Could you try `32` or something?
I don't expect formatter so much but I want to reduce diff to look history easily later.